jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/ProcessServiceImplPerProcessInstanceTest.java

```diff
@@ -16,89 +16,58 @@
 
 package org.jbpm.kie.services.test;
 
-import java.io.File;
-import java.io.FileOutputStream;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-import org.drools.compiler.kie.builder.impl.InternalKieModule;
 import org.drools.core.command.runtime.process.GetProcessInstanceCommand;
 import org.jbpm.kie.services.impl.KModuleDeploymentUnit;
-import org.jbpm.kie.test.util.AbstractKieServicesBaseTest;
 import org.jbpm.services.api.ProcessInstanceNotFoundException;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.BeforeClass;
+import org.jbpm.services.api.model.DeploymentUnit;
+import org.jbpm.test.services.AbstractKieServicesTest;
 import org.junit.Test;
-import org.kie.api.KieServices;
-import org.kie.api.builder.ReleaseId;
 import org.kie.api.runtime.process.ProcessInstance;
 import org.kie.api.runtime.process.WorkItem;
 import org.kie.internal.KieInternalServices;
 import org.kie.internal.process.CorrelationKey;
 import org.kie.internal.runtime.conf.RuntimeStrategy;
 import org.kie.internal.runtime.manager.context.ProcessInstanceIdContext;
-import org.kie.scanner.KieMavenRepository;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import static org.junit.Assert.*;
-import static org.kie.scanner.KieMavenRepository.getKieMavenRepository;
 
-public class ProcessServiceImplPerProcessInstanceTest extends AbstractKieServicesBaseTest {
+public class ProcessServiceImplPerProcessInstanceTest extends AbstractKieServicesTest {
```

Comment:
Why not to extend `AbstractKieServicesBaseTest` and override desired methods? GAV is the same so we won't need to specify it here I think, since it is already defined in `*BaseTest` class.

Reply:
the whole point here was to simulate how end user would make use of this AbstractKieServicesTest class so to make sure only these two (optionally three) methods needs to be implemented.

Reply:
Ah, OK. So AbstractKieServicesBaseTest is only "our" approach, but user is encouraged to use directly AbstractKieServicesTest in case he doesn't have so complex test hierarchy as we have?

Reply:
yup